### PR TITLE
UI test for BubbleChoice level

### PR DESCRIPTION
**Pull request by maddiedierker**

### CODE REVIEW

File: dashboard/config/scripts/allthethings.script [hunk 1]
```diff
@@ -1,5 +1,6 @@
 hideable_stages true
 stage_extras_available true
+family_name ''
```
Comment (maddiedierker) [inline dashboard/config/scripts/allthethings.script:3]: note: when i modified `allthethings.script` to add a new stage/level, some properties we've updated were pulled in as well (just the addition of `family_name ''` and how we label assessments)

File: dashboard/config/scripts/allthethings.script [hunk 2]
```diff
@@ -247,13 +248,16 @@ level 'allthethings HOC Dance 1'
...
-assessment 'allthethings-mini-rubric'
-assessment 'allthethings-mini-rubric-weblab'
-assessment 'allthethings-mini-rubric-gamelab'
+level 'allthethings-mini-rubric', assessment: true
+level 'allthethings-mini-rubric-weblab', assessment: true
+level 'allthethings-mini-rubric-gamelab', assessment: true
 
 stage 'Pluralsight Test', lockable: true
-assessment 'Pluralsight LevelGroup Test'
+level 'Pluralsight LevelGroup Test', assessment: true
```
Comment (Erin007) [inline dashboard/config/scripts/allthethings.script:257]: Thanks for updating all of these along the way.

**Review by Erin007 — APPROVED** [review]

**Review by davidsbailey — APPROVED** [review]

**Comment by davidsbailey** [comment]
sorry to see that this remains a drone-only repro :-(

 to iterate more quickly, the following things can help:
1. skip unit tests
2. tell drone to run only your UI test (and no eyes tests) by modifying this code: https://github.com/code-dot-org/code-dot-org/blob/945918cb1a7a39625eb16f2367a425f790d118f4/lib/rake/circle.rake#L102-L106
3. create a new branch / PR specifically for drone runs, to help you make sure you don't accidentally merge the above to staging

alternately, if you have any trouble running drone "locally" one idea is to run it on a dev EC2 instance where you might run into fewer problems since it is running on linux.

**Comment by davidsbailey** [comment]
another idea (not faster, but might fix things) is to change this line to use `rake seed:all` instead: https://github.com/code-dot-org/code-dot-org/blob/c1b85cf3e159eb11613765575e70deb8b0d3d008/.circleci/config.yml#L186

**Comment by maddiedierker** [comment]
@davidsbailey thank you, those are all really helpful tips! how do i skip unit tests? thankfully, i've never had to do that before 😆

**Comment by davidsbailey** [comment]
just include `[skip unit]` in the most recent commit name. other similar tags are listed at the top of circle.rake.

**Comment by davidsbailey** [comment]
> another idea (not faster, but might fix things) is to change this line to use `rake seed:all` instead:
> 
> https://github.com/code-dot-org/code-dot-org/blob/c1b85cf3e159eb11613765575e70deb8b0d3d008/.circleci/config.yml#L186

actually, this is where you'd want to use `rake seed:all`: https://github.com/code-dot-org/code-dot-org/blob/945918cb1a7a39625eb16f2367a425f790d118f4/lib/rake/circle.rake#L163

otherwise the unit test container might try to seed (unclear which wins if you also try to skip unit tests).

**Pull request merged**